app/src/main/java/com/github/se/signify/model/navigation/NavigationActions.kt

```diff
@@ -6,7 +6,8 @@ import com.github.se.signify.model.auth.UserSession
 
 open class NavigationActions(
     private val navController: NavHostController,
-    private val userSession: UserSession
+    private val userSession: UserSession,
+    var tutorialActive: Boolean = false
```

Comment:
Do you still need this after blocking the user from interacting with screens?
If not, you should revert this change, since we will want the tutorial to make the user navigate between screens.
Also, this clutters the `NavigationActions` class, and is error prone, as you could forget to unset the `tutorialActive` flag, and break the app.

Reply:
Absolutely, this is not necessary now after blowing the user's interaction. I will revert this now. Thank's for pointing out this.